src/common/intl/locales/fr.js

```diff
@@ -3051,44 +3050,44 @@ export default {
   // Original text: "address[:port]"
   serverPlaceHolderAddress: 'adresse[:port]',
 
-  // Original text: 'label'
-  serverPlaceHolderLabel: undefined,
+  // Original text: "label"
+  serverPlaceHolderLabel: 'nom',
 
   // Original text: "Connect"
   serverConnect: 'Connecter',
 
-  // Original text: 'Error'
-  serverError: undefined,
+  // Original text: "Error"
+  serverError: 'Erreur',
 
-  // Original text: 'Adding server failed'
-  serverAddFailed: undefined,
+  // Original text: "Adding server failed"
+  serverAddFailed: "Echec de l'ajout du serveur",
 
-  // Original text: 'Status'
-  serverStatus: undefined,
+  // Original text: "Status"
+  serverStatus: 'Statut',
 
-  // Original text: 'Connection failed. Click for more information.'
-  serverConnectionFailed: undefined,
+  // Original text: "Connection failed. Click for more information."
+  serverConnectionFailed: "Echec de connexion. Cliquer pour plus d'informations",
 
-  // Original text: 'Connecting...'
-  serverConnecting: undefined,
+  // Original text: "Connecting..."
+  serverConnecting: 'Connexion...',
 
-  // Original text: 'Connected'
-  serverConnected: undefined,
+  // Original text: "Connected"
+  serverConnected: 'Connecté',
 
-  // Original text: 'Disconnected'
-  serverDisconnected: undefined,
+  // Original text: "Disconnected"
+  serverDisconnected: 'Déconnecté',
 
-  // Original text: 'Authentication error'
-  serverAuthFailed: undefined,
+  // Original text: "Authentication error"
+  serverAuthFailed: "Erreur d'authentification",
 
-  // Original text: 'Unknown error'
-  serverUnknownError: undefined,
+  // Original text: "Unknown error"
+  serverUnknownError: 'Erreur inconnue',
 
-  // Original text: 'Invalid self-signed certificate'
-  serverSelfSignedCertError: undefined,
+  // Original text: "Invalid self-signed certificate"
+  serverSelfSignedCertError: 'Certificat auto-signé invalide',
```

Comment:
The original text is not perfect, the self-signed certificate is not invalid, the certificate is because it is self-signed.

Maybe *Certificat auto-signé rejeté* would be better.